package.json

```diff
@@ -104,7 +104,7 @@
     "@babel/register": "7.4.4",
     "@elastic/charts": "^6.0.1",
     "@elastic/datemath": "5.0.2",
```

Comment:
Would you mind removing line 83 as well please?

Reply:
Ah yep